documentation/user-interface/fixture-editor.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: docs
+title: Fixture Editor
+permalink: /documentation/user-interface/fixture-editor.html
+---
+# Fixture Editor
+
+The **Fixture** editor allows you to view and modify a fixture. The editor opens when you double-click the fixture in the [Explorer](explorer-panel.md) panel.
+
+The editor consists of the following parts:
+
+* The toolbar - Provides information about the fixture (the name and tested webpage's URL) and buttons for performing [common tasks](#common-tasks).
+
+    You can click the webpage's URL to open the webpage in the browser.
+
+* The test list - Displays tests included to the fixture. You can click a test's row to open the test in the [Test](test-editor.md) editor.
+
+![Fixture editor](../../images/user-interface/fixture-editor.png)
+
+## Common Tasks
+
+You can perform the following tasks in the editor:
+
+* [Record New Tests](#record-new-tests)
+* [Run the Fixture or Individual Tests](#run-the-fixture-or-individual-tests)
+* [Convert the Fixture to JavaScript](#convert-the-fixture-to-JavaScript)
+* [Modify the Fixture's Properties](#modfy-the-fixtures-properties)
+* [Delete Tests](#delete-tests)
+
+### Record New Tests
+
+Click the ![Record button](../../images/working-with-testcafe-studio/record-test-icon.png) **Record a new test** button on the editor's toolbar to start recording a new test and add it to the fixture.
+
+### Run the Fixture or Single Test
+
+Click the ![Run tests button](../../images/working-with-testcafe-studio/action-run-icon.png) **Run all tests** button on the editor's toolbar to run the entire fixture.
+
+To run a single test, hover the mouse cursor over the test's row and click the ![Run test button](../../images/working-with-testcafe-studio/action-run-icon.png) **Run test** button displayed to the right of it.
+
+### Convert the Fixture to JavaScript
+
+Click the ![Convert icon](../../images/user-interface/js-icon.svg) **Convert to JavaScript** button on the editor's toolbar to convert the fixture to JavaScript. The **Convert to JavaScript** dialog appears and allows you to specify a .js file's name to be created.
```

Comment:
looks like a name is to be created

I would write `the name of the .js file to be created` but this is not accepted now.

 Maybe `to specify the created .js file's name.`?